src/psyclone/psyir/frontend/fparser2.py

```diff
@@ -5327,23 +5226,17 @@ def _process_args(self, node, call, canonicalise=None):
         all optional arguments as named arguments, which would result
         in sum(a, dim=dim, mask=mask) in this case.
 
-        :param node: an fparser call node representing a call or \
+        :param node: an fparser call node representing a call or
             an intrinsic call.
-        :type node: :py:class:`fparser.two.Fortran2003.Call_Stmt` or \
-            :py:class:`fparser.two.Fortran2003.Intrinsic_Function_Reference`
-        :param call: a PSyIR call argument representing a call or an \
+        :param call: a PSyIR call argument representing a call or an
             intrinsic call.
-        :type call: :py:class:`psyclone.psyir.nodes.Call` or \
-            :py:class:`psyclone.psyir.nodes.IntrinsicCall`
-        :param function canonicalise: a function that canonicalises \
-            the call arguments.
+        :param canonicalise: whether to canonicalise the call (for
```

Comment:
Can this argument be removed now?

Reply:
Renamed, and comment and docstring updated.